calendar/calendar-tool/tool/src/webapp/vm/calendar/chef_calendar_view.vm

```diff
@@ -22,3 +22,38 @@
 <script src="/library/webjars/fullcalendar/5.8.0/main.min.js"></script>
 <script src="/library/webjars/fullcalendar/5.8.0/locales-all.min.js"></script>
 <script src="/sakai-calendar-tool/js/sakai-calendar.js"></script>
+
+<script>
+  // This logic is associated to set the default subview, by day, month, week or list.
+  const defaultSubview = '$defaultSubview';
+  switch (defaultSubview) {
+    case 'day':
+      calendar.changeView('timeGridDay');
+      break;
+    case 'month':
+      calendar.changeView('dayGridMonth');
+      break;
+    case 'list':
+      calendar.changeView('listWeek');
+      break;
+    case 'week':
+    default:
+      calendar.changeView('timeGridWeek');
+      break;
```

Comment:
```suggestion
```

Reply:
It's a common pattern I use, do you think it harms?

Reply:
Okay, I'll include it.

Reply:
No, it doesn't do any harm. I'm just being pedantic.

Reply:
No problem, I've included the change.